packetbeat/_meta/kibana/6/dashboard/Packetbeat-cassandra.json

```diff
@@ -1,939 +1,1100 @@
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "index": "packetbeat-*",
+            "query": {
+              "query_string": {
+                "analyze_wildcard": true,
+                "query": "*"
+              }
+            }
+          }
+        },
+        "savedSearchId": "eaa83e60-190b-11e9-be0d-adde5066235e",
+        "title": "Cassandra ResponseType [Packetbeat]",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [
+            {
+              "enabled": true,
+              "id": "1",
+              "params": {},
+              "schema": "metric",
+              "type": "count"
+            },
+            {
+              "enabled": true,
+              "id": "2",
+              "params": {
+                "field": "cassandra.response.result.type",
+                "order": "desc",
+                "orderBy": "1",
+                "size": 10
+              },
+              "schema": "segment",
+              "type": "terms"
+            }
+          ],
+          "listeners": {},
+          "params": {
+            "addLegend": true,
+            "addTooltip": true,
+            "isDonut": false,
+            "legendPosition": "right",
+            "shareYAxis": true
+          },
+          "title": "Cassandra ResponseType",
+          "type": "pie"
+        }
+      },
+      "id": "Cassandra-ResponseType",
+      "type": "visualization",
+      "updated_at": "2019-01-15T21:27:07.580Z",
+      "version": 2
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "index": "packetbeat-*",
+            "query": {
+              "query_string": {
+                "analyze_wildcard": true,
+                "query": "*"
+              }
+            }
+          }
+        },
+        "savedSearchId": "eaa83e60-190b-11e9-be0d-adde5066235e",
+        "title": "Cassandra ResponseTime [Packetbeat]",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [
+            {
+              "enabled": true,
+              "id": "1",
+              "params": {
+                "field": "event.duration",
+                "percents": [
+                  5,
+                  25,
+                  50,
+                  75,
+                  95
+                ]
+              },
+              "schema": "metric",
+              "type": "percentiles"
+            },
+            {
+              "enabled": true,
+              "id": "2",
+              "params": {
+                "customInterval": "2h",
+                "drop_partials": false,
+                "extended_bounds": {},
+                "field": "@timestamp",
+                "interval": "auto",
+                "min_doc_count": 1,
+                "time_zone": "America/New_York",
```

Comment:
Can tz info here be problematic?

Reply:
I'm not at all sure why this got added or what effect it has. It looks like it is using a custom date histogram interval. I'll try switching that to automatic and see if this TZ goes away.

Reply:
I edited the object to remove the `time_zone` and the dashboard still works OK. Pushed an updated.